deployment_scripts/deploy_windows.ps1

```diff
@@ -115,12 +108,6 @@ function Deploy-Windows([String] $monkey_home = (Get-Item -Path ".\").FullName,
     "Removing zip file"
     Remove-Item $TEMP_OPEN_SSL_ZIP
 
-    # Download and install C++ redistributable
```

Comment:
We don't need this?

Reply:
Seemed to work without it ( I uninstalled )

Reply:
Turned out we need it for mongodb